shell.nix

```diff
@@ -1,9 +1,7 @@
-{ pkgs ? import <nixpkgs> { } }:
-
+{pkgs ? import <nixpkgs> {}}:
```

Comment:
same here, spaces improve readability but alejandra just randomly decided to remove the first and last spaces while keeping the ones around the question mark.